roles/ara_api/vars/CentOS_8.yaml

```diff
@@ -41,3 +41,5 @@ ara_api_mysql_packages:
   - redhat-rpm-config
   - python36-devel
   - gcc
+
+ara_api_python_command: /usr/bin/python3.9
```

Comment:
This should fix our issue with the jobs based on CentOS Stream 8 :+1: https://github.com/ansible-community/ara-collection/pull/71